### ISSUE TITLE
Chrome push test does not display push notification

### ISSUE THREAD

**Issue by bo-brainlabs** [GoogleChrome/push-notifications#41]
I've followed the tutorial up to the "Handle a push event section".
At the end of this section, I click the chrome devtools "Push" button, but no push notification is displayed on screen.
The service worker "sw.js" is executed as I can see the console log output and it reads:

```
[Service Worker] Push Received.
[Service Worker] Push had this data: "Test push message from DevTools."
```

But no push notification.
I'm using a 2013 macbook air running 10.12.3 & Chrome version 57

Is this just something I've done wrong or is this a common problem?

**Comment by gauntface** [comment]
I've not come across anyone experiencing this before. Maybe try unregistering the service worker and refreshing the page and going through it again, in case the service worker is just out of date.

**Comment by bo-brainlabs** [comment]
Hi @gauntface 
I found that refreshing it didn't seem to work, but using your demo site https://gauntface.github.io/simple-push-demo/ works like a charm...

Actually pasting that link in the terminal created a push notification and displayed all the test push notifications that didn't previously work...

So it's almost as if the notifications were waiting to be triggered by something else.
Any ideas?

I'll keep digging and hopefully find the problem when I fully understand how the service worker is doing it's thing.

**Comment by bo-brainlabs** [comment]
Thanks!

**Comment by albanx** [comment]
Having same issue following the same tutorial. The push button from Chrome dev tools does not work

**Comment by vtkhanh** [comment]
I got the same problem now :(

**Comment by portal7** [comment]
Same issue, Button "Push" from DevTools not works  ("focus" link button did not too)

**Comment by zenghj** [comment]
Same issue :(

**Comment by mqtsuo02** [comment]
This issue has been reported as a bug in Developer Tools.
https://github.com/GoogleChromeLabs/web-push-codelab/issues/49

**Comment by GitPawel** [comment]
Guys! It works on Google Canary:
I have: v. 73.0.3628.0 (official version) canary (32-bits)

**Comment by factcondenser** [comment]
Hopefully this helps someone else who googles their way to this thread:

[Notification.tag](https://developer.mozilla.org/en-US/docs/Web/API/Notification/tag) seems to behave differently between Firefox and Chrome. To see what I mean, visit this supremely helpful [push notification examples site](https://web-push-book.gauntface.com/demos/notification-examples/) (thanks for creating that, @gauntface!) in Firefox and click the first demo button entitled 'Example Notification'. Close the notification that appears and click the button again. You should receive a second notification. Now, repeat that process in Chrome. Unless I'm missing something, you should only have received one notification that appears after the first click. Subsequent clicks don't cause more notifications to appear. Removing the `tag` entry from the options hash ([line 42 of notification-examples.js](https://github.com/gauntface/web-push-book/blob/130297278d2222d4dd01b48a1d42e4575b32f41e/src/demos/notification-examples/notification-examples.js#L42)) causes the button to behave the same way in both Firefox and Chrome.

In conclusion, if you, like me, were confused as to why your push notifications weren't showing up in Chrome, check whether you're passing a tag in the options hash to `RegistrationServiceWorker.showNotification()`.

**Comment by danduongfr** [comment]
Had the same issue on Apple MacPro.
Had to authorize notification for Chrome in Apple System Preferences.

![Screen Shot 2020-10-29 at 3 09 57 PM](https://user-images.githubusercontent.com/23695307/97537071-29934300-19f9-11eb-8df0-4036693da600.png)
